framework/Element.js

```diff
@@ -44,22 +44,47 @@ export default class Element {
         }
     }
 
-    async click() {
+    async getCoordinates(xOffset, yOffset, elementHandle) {
+        const rect = await elementHandle.boundingBox();
+        const x = xOffset !== null ? xOffset : rect.width / 2;
+        const y = yOffset !== null ? yOffset : rect.height / 2;
+        const xCoordinate = rect.x + x;
+        const yCoordinate = rect.y + y;
+        console.log(`Action on page at position x: ${xCoordinate}, y: ${yCoordinate}`);
+        console.log(`Action on element rectangle at position x: ${x}, y: ${y}`);
+        return [xCoordinate, yCoordinate];
```

Comment:
For readability I suggest returning this as an object, like `return { x: xCoordinate, y: yCoordinate };`
Then it can be called as `coordinates.x`, `coordinates.y` in line 63 and others

Reply:
Addressed